src/main/java/core/basesyntax/service/fileservice/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.fileservice;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FILE_PATH = "src\\main\\resources\\report.csv";
```

Comment:
Wouldn't it be better to put the file path in the main method? I don't think this is a mistake, but still.

Reply:
done